src/index.html

```diff
@@ -7,6 +7,51 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__row calendar__row--1">
```

Comment:
do not create a separate `div` for each row
you should define position only via CSS

in HTML should be only `calendar` div that contains 31 cell divs with `calendar__cell` class (check task [description](https://github.com/mate-academy/layout_calendar#task))

If you need any help or have questions - feel free to ask in chat or join to the next HTML/CSS Q&A session